Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.h

```diff
@@ -63,7 +63,11 @@ struct VariationDefaults {
 
 typedef HashMap<FontTag, VariationDefaults, FourCharacterTagHash, FourCharacterTagHashTraits> VariationDefaultsMap;
 
-RetainPtr<CTFontRef> preparePlatformFont(UnrealizedCoreTextFont&&, const FontDescription&, const FontCreationContext&, bool applyWeightWidthSlopeVariations = true);
+enum class ApplyWeightWidthSlopeVariations : uint8_t {
+    Yes,
+    No,
+};
```

Comment:
```suggestion
enum class ApplyWeightWidthSlopeVariations : bool {
    No,
    Yes,
};
```

Aside from this nit, the name `ApplyWeightWidthSlopeVariations` feels a bit clunky, I don't have any better suggestions, so I'm fine landing as is.

Maybe consider simplifying the name by transforming to something like `SlopeVariations::{ None, WeightWidth }` (not sure if it makes sense with the current code, but maybe you have better ideas).

Reply:
Yeah, this means "apply the weight variations, width variations, and the slope variations"

Probably "traits variations" is better